src/OrchardCore.Modules/OrchardCore.Media/Startup.cs

```diff
@@ -151,6 +151,9 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IContentPartFieldDefinitionDisplayDriver, MediaFieldSettingsDriver>();
 
             services.AddRecipeExecutionStep<MediaStep>();
+
+            // MIME types
+            services.AddSingleton<IContentTypeProvider, FileExtensionContentTypeProvider>();
```

Comment:
Should be a `TryAdd` in case another library registers one already. i am doing the change.